docs/assets/overrides/partials/copyright.html

```
@@ -0,0 +1,22 @@
+    <div class="md-copyright__highlight">
+        {{ config.copyright }}
+    </div>
+    <a href="{{ config.repo_url }}/blob/main/LICENSE">Apache-2.0 LICENSE</a>
+    {% endif %}
+</div>
+
+<div class="md-copyright">
+    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">Made with Material for MkDocs</a>
+    | <a href="https://www.networktocode.com/community/" target="_blank" rel="noopener">Join Nautobot Slack</a>
+    {% if config.extra.ntc_sponsor == true %}
+    | <a href="https://networktocode.com" target="_blank" rel="noopener">Sponsored by <img
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)